### PR TITLE
update openjdk base image

### PR DESCRIPTION
**Pull request by vroldanbet**
update openjdk base image

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM registry.opensource.zalan.do/stups/openjdk:8-52
+FROM registry.opensource.zalan.do/stups/openjdk:8-cd28
```
Comment (harti2006) [inline Dockerfile]: I would recommend you to use the "latest" tag here.

**Comment by vroldanbet** [comment]
@harti2006 but we want reproducible builds, right? :dancer:

**Comment by harti2006** [comment]
Of course you might lose the ability of reproducible builds, but you will save a lot of time ;-) That's the tradeoff.

**Comment by vroldanbet** [comment]
:+1:

**Comment by harti2006** [comment]
:+1:

**Pull request merged**